VideoWeb/VideoWeb/ClientApp/src/app/services/conference/video-control-cache-local-storage.service.ts

```diff
@@ -29,7 +29,7 @@ export class VideoControlCacheLocalStorageService implements IVideoControlCacheS
         state[conferenceId] = hearingControlStates;
 
         this.localStorageService.save(this.localStorageKey, state);
-        return true;
+        return of(void 0);
```

Comment:
What's the purpose of this change? Could the method not return anything instead? Or could we set the return type as an observable?

Reply:
this returns observable void